lib/src/Uncertainty/Algorithm/OrthogonalBasis/StandardDistributionPolynomialFactory.cxx

```diff
@@ -196,6 +196,12 @@ void StandardDistributionPolynomialFactory::checkSpecificFamily()
     referenceFamily = KrawtchoukFactory(static_cast<UnsignedInteger>(parameter[0]), parameter[1]);
     hasClassMatch = true;
   }
+  if (measureType == "Bernoulli")
```

Comment:
The "switch" C++ statement would be more appropriate, isn't it ?

Reply:
Doesn't work with complex types, only simple types (int, char & co)